geps/gep-1709.md

```diff
@@ -97,25 +92,37 @@ conformance suite as a library.
 
 ### Profiles
 
-Initially there will be two profiles:
-
-- `Layer4`
-- `Layer7`
-
-> **NOTE**: these are simply the initial profiles we're going to start with,
-> it's plausible for there to be more in the future.
-
-These named profiles are effectively categories which represent the high level
-grouping of tests. When conformance is reported using one of these profiles
-extra features can be covered according to support levels:
+"Profiles" are effectively categories which represent the high level grouping of
+tests related to some feature (or feature set) of Gateway API. When conformance
+is reported using one of these profiles extra features can be covered according
+to support levels:
 
+- `core`
 - `extended`
 
 > **NOTE**: `implementation-specific` doesn't really have much in the way of
 > tests today, but it is something users want to be able to display. We leave
 > door open for it later and mention it in the [alternatives
 > considered](#alternatives-considered) section below.
 
+We will start with the following named profiles:
+
+- `HTTPRoute`
+- `TLSRoute`
+
+These profiles correspond with `*Route` type APIs that we currently have tests
```

Comment:
where would the existing tests corresponding to Gateway/GatewayClass validation/features fall here? would they also automatically be run?

Reply:
That's a good point. Implicitly we understand they would run under the route named profiles, but we didn't actually explicitly spell that out yet. f9d04b965ece6074d52ae3988f8fb4cc704c1e53 takes care of adding a note about this :+1: